packages/circuits/helpers/ascii.circom

```diff
@@ -1,19 +0,0 @@
-pragma circom 2.1.4;
-include "circomlib/circuits/comparators.circom";
-
-// from nozee (emmaguo13, sehyunc, kaylee)
-// https://github.com/emmaguo13/zk-blind/blob/master/circuits/ascii.circom
-// only converts ascii to numbers from 0-9
-template AsciiToNum (max_input) {
```

Comment:
This file is imported from Nozee so shouldn't be deleted, but moved to string utils

Reply:
> ascii.circom was removed. Moved and renamed the similar circuit from email wallet to utils/bytes.circom (DigitBytesToInt)

We already had a similar circuit in email wallet which was imported here. So this one was deleted.